mitiq/cdr/clifford_training_data.py

```diff
@@ -86,8 +86,6 @@ def generate_training_circuits(
             if not cirq.has_stabilizer_effect(op)
         ]
     )
-    if len(non_clifford_indices_and_ops) == 0:
-        raise ValueError("Circuit is already Clifford.")
```

Comment:
Ah yes I see. Perhaps it's worth altering the functionality of clifford_training_data.py to allow for Clifford circuit input too? Instead of raising a ValueError, we could return a list of the original circuit `n` times, where `n` is the number of training circuits. What do you think?

Reply:
Sounds good!